src/components/Addbook.js

```diff
@@ -1,50 +1,108 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './navbar.css';
 
 function AddBook() {
+  const [newbook, setNewbook] = useState({
+    title: '',
+    author: '',
+    category: 'Category',
+  });
+
+  const [books, setBooks] = useState([]);
+
+  function AddbookHundle(e) {
+    e.preventDefault();
+
+    const newBook = {
+      id: crypto.randomUUID(),
+      title: newbook.title,
+      author: newbook.author,
+      category: newbook.category,
+    };
+
+    setBooks((currentBooks) => [...currentBooks, newBook]);
+
+    setNewbook({
+      title: '',
+      author: '',
+      category: 'Category',
+    });
+  }
+
+  function DeleteBook(id) {
+    setBooks((currentBooks) => currentBooks.filter((book) => book.id !== id));
+  }
+
   return (
     <>
-      <div className="bookcont">
-        <div className="books-info">
-          <span className="book-cata">Action</span>
-          <h3 className="book-title">The Hunger Games</h3>
-          <p className="book-author">Suzanne Collins</p>
-          <button type="button" className="btn" id="commit">
-            Comments
-          </button>
-          <button type="button" className="btn" id="remove">
-            Remove
-          </button>
-          <button type="button" className="btn" id="edit">
-            Edit
-          </button>
-        </div>
-        <div className="compelet">
-          <span className="precent">64%</span>
-          <h5>completed</h5>
-        </div>
-        <div className="current-part">
-          <p className="current-chapter">CURRENT CHAPTER</p>
-          <h4>Chapter 17</h4>
-          <button type="button" className="upd-btn">
-            UPDATE PROGRESS
-          </button>
-        </div>
+      <div className="book-list">
+        {books.map((book) => (
+          <div className="bookcont" key={book.id}>
+            <div className="books-info">
+              <span className="book-cata">{book.category}</span>
+              <h3 className="book-title">{book.title}</h3>
+              <p className="book-author">{book.author}</p>
+              <button type="button" className="btn" id="commit">
+                Comments
+              </button>
+              <button
+                type="button"
+                className="btn"
+                id="remove"
+                onClick={() => DeleteBook(book.id)}
+              >
+                Remove
+              </button>
+              <button type="button" className="btn" id="edit">
+                Edit
+              </button>
+            </div>
+            <div className="compelet">
+              <span className="precent">
+                64%
+              </span>
```

Comment:
Just as the previous reviewer requested, kindly break the `addBook` component into separate components and ensure they are placed in separate files.

for instance, this `AddBook` component can contain the form for adding new books and be responsible for adding new books😉.

then you can create another component `Book` which should be in a separate file `Book.js`. this should take the book data as a prop and render the title and author of a book and has a button for removing books😉.
Lastly, another component called the `bookList` in a `bookList.js` file. this should be responsible for rendering all the books in the array😉.

I understand you have implemented these features in this class, but breaking your project into smaller components is important for component reusability and separation of concerns😉. This makes your codebase readable and maintainable👍🏼.